README.md

```diff
@@ -1,2 +1,7 @@
 # ML-python-sklearn-rc-cpu-service
-Jenkins X quick start for training and deploying Ridge Classification using the SciKit-Learn library
+
+Quickstart project for executing a wine classifier using the SciKit-Learn framework on a CPU.
+
+Running pip install requirements.txt and then python `app.py` will start the app on localhost where the user can send GET requests to perform inference.
```

Comment:
Put `pip` and `python` commands in \` \` quotes so markdown renders them properly